lib/gems/pending/appliance_console/cli.rb

```diff
@@ -117,6 +122,7 @@ def parse(args)
         opt :username, "Database Username",  :type => :string,  :short => 'U', :default => "root"
         opt :password, "Database Password",  :type => :string,  :short => "p"
         opt :dbname,   "Database Name",      :type => :string,  :short => "d", :default => "vmdb_production"
+        opt :db_hourly_maintenance, "Configure database hourly maintenance", :type => :bool, :short => 'b'
```

Comment:
I don't think a short option here makes sense.

What we're doing is a bit too complicated for a once character abbreviation I think.